gradio/components.py

```diff
@@ -2570,6 +2570,13 @@ def style(self):
         return self
 
 
+class Variable(State):
```

Comment:
I think we need to fix the config for Variable to display `state` as the type. As is, it shows Variable and it causes demos that use `Variable` to break.

Reply:
Good catch!